### DISCUSSION TITLE
Custom chart legend

### DISCUSSION THREAD

**Discussion by andrewleith** [Budibase/budibase#4274] [Ideas]
The chart legends are based on my column names and as such aren't really UI-friendly.  

Edit: ~As a side note, I tried to rename them with spaces in the names in the underlying table and it dropped my columns entirely :(~ I think it just pushed the column off of my small laptop screen and I didn't realize it! :)

Is it possible to rename these somehow in the UI? 

Thanks!

**Comment by mjashanks** [comment]
It's not possible at the moment. There are quite a few improvements that we'd like to make to charts actually. 

Thanks for the feedback - sorry I can't give you a more positive answer!
